src/braket/device_schema/quera/quera_ahs_paradigm_properties_v1.py

```diff
@@ -124,21 +124,181 @@ class PerformanceLattice(BaseModel):
+    atomCaptureProbabilityTypical: Annotated[Decimal, Field(ge=0, le=1)]
+    atomCaptureProbabilityWorst: Annotated[Decimal, Field(ge=0, le=1)]
+    atomDetectionErrorFalsePositiveTypical: Annotated[Decimal, Field(ge=0, le=1)]
+    atomDetectionErrorFalsePositiveWorst: Annotated[Decimal, Field(ge=0, le=1)]
+    atomDetectionErrorFalseNegativeTypical: Annotated[Decimal, Field(ge=0, le=1)]
+    atomDetectionErrorFalseNegativeWorst: Annotated[Decimal, Field(ge=0, le=1)]
+
+
+class RabiCorrection(BaseModel):
+    """
+    Correction factors for calculating the fraction of the expected Rabi oscillation frequency as a
+        function of ramp time, in the absence of any local detuning pattern.
+    Attributes:
+        rampTime (Decimal): The ramp time. (measured in s)
+        rabiCorrection (Annotated[Decimal, Field(ge=0.0, le=1.0)]): The fraction of the expected
+            rabi oscillation frequency. Normalized to the range [0.0, 1.0].
+    """
+
+    rampTime: Decimal
+    rabiCorrection: Annotated[Decimal, Field(ge=0.0, le=1.0)]
 
 
 class PerformanceRydbergGlobal(BaseModel):
     """
     Parameters determining the limitations of the global driving field
     Attributes:
-        rabiFrequencyErrorRel (Decimal): random error in the Rabi frequency, relative (unitless)
+        rabiFrequencyErrorRel (Decimal): Total error in the Rabi frequency due to inhomogeneity and
+            variations in time, relative. (unitless)
+        rabiFrequencyGlobalErrorRel (Decimal): RMS Rabi frequency variation in time as a relative
+            value. (unitless)
+        rabiFrequencyInhomogeneityRel (Decimal): RMS Rabi frequency inhomogeneity over the user
+            region, relative. (unitless)
+        groundDetectionError (Decimal): Probability of mis-detecting a ground-state atom as a
+            Rydberg-state atom. (unitless)
+        rydbergDetectionError (Decimal): Probability of mis-detecting a Rydberg-state atom as a
+            ground-state atom. (unitless)
+        groundPrepError (Annotated[Decimal, Field(ge=0, le=1)]): Probability of failing to
+            initialize an atom in the ground state
+            prior to user-programmed coherent evolution, in the absense of any local detuning
+            pattern. Normalized to 1.
+        rydbergPrepErrorBest (Annotated[Decimal, Field(ge=0, le=1)]): Probability of failing to
+            initialize an atom in the Rydberg state by an optimal (for that site) user specified
+            pi-pulse from the ground state at maximum Rabi frequency, in the absense of any local
+            detuning pattern. Normalized to 1.
+        rydbergPrepErrorWorst (Annotated[Decimal, Field(ge=0, le=1)]): Worst-case probability of
+            failing to initialize an atom in the Rydberg state by a user specified pi-pulse from
+            the ground state at maximum Rabi frequency, optimized for a different site, in the
+            absense of any local detuning pattern. Normalized to 1.
```

Comment:
nit: This `absence` needs to corrected in multiple places?

Reply:
Yep, addressed